examples/cluster/mortgage-runner.py

```diff
@@ -0,0 +1,46 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+# pip install git+https://github.com/intel-go/ibis.git@develop
+
+# NOTE: expects https://github.com/intel-go/omniscripts/ checked out and in PYTHONPATH
+
+# the following import turns on experimental mode in Modin,
+# including enabling running things in remote cloud
+import modin.experimental.pandas as pd  # noqa: F401
+from modin.experimental.cloud import create_cluster, get_connection
+
+from mortgage import run_benchmark
+from mortgage.mortgage_pandas import etl_pandas
+
+test_cluster = create_cluster("aws", "aws_credentials", cluster_name="rayscale-test", region="eu-north-1", zone="eu-north-1b", image="ami-00e1e82d7d4ca80d3")
+with test_cluster:
+    conn = get_connection()
+    np = conn.modules["numpy"]
+    etl_pandas.__globals__["np"] = np
```

Comment:
is this still needed?

Reply:
In this PR - yes.

I've splitted my work into several atomic pieces, and automagic numpy wrapper PR (which depends on this one) has not yet been filed.

Reply:
Will be tracked by this: https://github.com/modin-project/modin/issues/1819